python/tchannel/statsd.py

```diff
@@ -91,6 +107,19 @@ def on_operational_error(self, request, error):
 
         self._statsd.count(key, 1)
 
+    def outbound_latency_per_attempt(self, request):
+        if request.tracing.span_id not in self.outbound_attempt:
+            return
+        (start_time, retry_count) = self.outbound_attempt.pop(
+            request.tracing.span_id)
+
+        latency_statsd_name = "tchannel.outbound.calls.per-attempt.latency"
+        key = common_prefix(latency_statsd_name, request) + '.' + str(
+            retry_count)
```

Comment:
Again, we shouldn't be emitting complete stat streams for each attempt count. You don't need to differentiate between the latencies of the different attempts. Just emit a latency metric for each attempt under the same metric name.